python/ray/serve/controller.py

```diff
@@ -674,21 +722,26 @@ async def _set_traffic(self, endpoint_name: str,
         traffic_policy = TrafficPolicy(traffic_dict)
         self.current_state.traffic_policies[endpoint_name] = traffic_policy
 
+        return_uuid = self._create_event_with_result({
+            endpoint_name: traffic_policy
+        })
         # NOTE(edoakes): we must write a checkpoint before pushing the
         # update to avoid inconsistent state if we crash after pushing the
         # update.
         self._checkpoint()
 
         self.notify_traffic_policies_changed()
+        return return_uuid
 
     async def set_traffic(self, endpoint_name: str,
-                          traffic_dict: Dict[str, float]) -> None:
+                          traffic_dict: Dict[str, float]) -> uuid4:
```

Comment:
the type should be uuid.UUID instead of the function uuid4

Reply:
Oh!! Thanks!